Sources/Views/SubscriptionConfirmation.swift

```diff
@@ -37,14 +37,155 @@ public func subscriptionConfirmation(
       subscriberState: subscriberState,
       referrer: referrer,
       episodeStats: episodeStats,
-      lane: lane
+      lane: lane,
+      coupon: coupon,
+      useRegionalDiscount: subscribeData.useRegionalDiscount
     ),
     currentUser.map { teamMembers(lane: lane, currentUser: $0, subscribeData: subscribeData) } ?? [],
     billingPeriod(coupon: coupon, lane: lane, subscribeData: subscribeData),
     currentUser != nil
       ? payment(lane: lane, coupon: coupon, stripeJs: stripeJs, stripePublishableKey: stripePublishableKey)
       : [],
-    total(isLoggedIn: currentUser != nil, lane: lane, coupon: coupon, referrer: referrer)
+    total(
+      isLoggedIn: currentUser != nil,
+      lane: lane,
+      coupon: coupon,
+      referrer: referrer,
+      useRegionalDiscount: subscribeData.useRegionalDiscount
+    )
+  )
+}
+
+private func additionalDiscountInfo(
```

Comment:
this function determines what to show in the yellow info box. the logic is pretty gnarly because there are 4 possibilities:

<img width="956" alt="image" src="https://user-images.githubusercontent.com/135203/81346072-7f041e00-906e-11ea-88ab-af69e5a3cd01.png">

<img width="945" alt="image" src="https://user-images.githubusercontent.com/135203/81346088-83c8d200-906e-11ea-9f1a-a15bbc60ea68.png">

<img width="955" alt="image" src="https://user-images.githubusercontent.com/135203/81346095-888d8600-906e-11ea-821e-db7de0ed762e.png">

<img width="959" alt="image" src="https://user-images.githubusercontent.com/135203/81346098-8cb9a380-906e-11ea-9f5d-9b8db9db06d2.png">